incubator/ado-agent/values.yaml

```diff
@@ -0,0 +1,20 @@
+replicaCount: 1
+image:
+  repository: gambtho/azure-pipeline-agent
+  tag: latest
+  pullPolicy: Always
+ingress:
+  enabled: false
+vsts:
+  account: "replacme"
+  token: "replaceme"
+  pool: "replaceme"
+service:
+  type: ClusterIP
+resources:
```

Comment:
Can you please comment out the resources. There are details in the review guidelines about it at https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration